lintreview/tools/credo.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import
+import os
+import logging
+import lintreview.docker as docker
+from lintreview.tools import Tool, process_quickfix
+
+log = logging.getLogger(__name__)
+
+
+class Credo(Tool):
+    """
+    Run credo on files.
+    """
+
+    name = 'credo'
+
+    def check_dependencies(self):
+        """
+        See if credo image exists
+        """
+        return docker.image_exists('credo')
+
+    def match_file(self, filename):
+        base = os.path.basename(filename)
+        name, ext = os.path.splitext(base)
+        return ext in ('.ex', '.exs')
+
+    def process_files(self, files):
+        """
+        Run code checks with credo.
+        """
+        log.debug('Processing %s files with %s', files, self.name)
+        credo_options = []
```

Comment:
With this list empty users will not be able to enable `--strict` or use the `exclude` and `only` options. Right now the loop on line 35-39 does nothing but log errors.